OpenOversight/app/templates/list_officer.html

```diff
@@ -12,16 +12,18 @@
         rel="stylesheet">
 {% endblock head %}
 {% block content %}
-  <div class="container" role="main">
-    <h1>{{ department.display_name }} Officers</h1>
+  <div class="container pb-5" role="main">
+    <div class="frontpage-leads">
+      <h1>{{ department.display_name }} Officers</h1>
+    </div>
     <div class="row">
       <div class="filter-sidebar col-sm-3">
         <h3 class="sidebar-title">Filter officers</h3>
         <form class="form" method="get" role="form">
           <div class="panel">
-            <div class="panel-heading{% if not form_data['first_name'] and not form_data['last_name'] %} collapsed{% endif %}"
```

Comment:
Should we keep the `if` here like we did below?

Reply:
Good catch! I think this is an artifact of cherry-picking from our instance since we always show the name panel